spec/test_track_spec.rb

```diff
@@ -65,16 +65,4 @@
       end
     end
   end
-
-  describe ".fully_qualified_cookie_domain_enabled?" do
```

Comment:
There's already coverage in session_spec.rb for when the flag is enabled, so i didn't think these tests added much.